solvers.tex

```diff
@@ -60,7 +60,7 @@
 >>> solveset(x - x, x, domain=S.Reals)
 (-oo, oo)
 >>> solveset(x - x, x, domain=S.Complexes)
-S.Complexes
```

Comment:
I see. I think we ought to use Python 3 printing as our default, now that we don't support Python 2.6 and we can use set literals. I opened https://github.com/sympy/sympy/pull/11116 for this.